Library/Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,70 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.9.6.tar.gz"
+  sha256 "cf2dad62e50c5c5cff88a4ad959af3aa402aa7e8aa1692bdf807052c0fe4aeb9"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  ALL_CLIENTS = [
```

Comment:
Feel free to have some of these clients always installed.  We try to select reasonable defaults and then offer a few options for things that many but not all users will want.